docker-compose.yml

```diff
@@ -21,4 +21,4 @@ services:
        - "80:80"
       environment:
        - NGINX_HOST=foobar.com
-       - NGINX_PORT=80
+       - NGINX_PORT=8080
```

Comment:
If the change of the NGINX_PORT from 80 to 8080 has any implications on other parts of the system or if it requires certain conditions to work properly, it would be beneficial to document this in a comment next to the changed line. [medium]